File: Level 026/homework/hw4.py
# 6) შექმენით სია რომელშიც იქნება სახელები შემდეგ კი შექმენით პროგრამა რომელიც ამოშლის ყველა სახელს რომელიც "t" ასოზე იწყება და ჩაამატებს ახალ სიაში

my_list = ["nika", "dachi", "tgiorgi", "tguga", "tnikusha"]

filtered_names = []

for name in my_list:
    if name[0] != "t":
        filtered_names.append(name)


print(filtered_names)


        

File: Level 071/homework/hw4.js
/* 5)Splice მეთოდის გამოყენებით ჩაანაცვლეთ სიაში ყველა უარყოფითი რიცხვი 0 - ით */

let my_list = [3, -7, 12, -2, 8, -10, 5, -1]

for (i = 0; i < my_list.length; i++) {
    if (my_list[i] < 0) {
        my_list.splice(i, 1, 0)
    }
}

console.log(my_list)